docs/managed-services.md

```diff
@@ -24,6 +22,9 @@ MySQL یکی از محبوب‌ترین RDBMS‌های امروزی است که
 |service_name| string| mysql| نامی که برای سرویس مایلید در نظر گرفته شود|
 |phpmyadmin_enabled|boolean | true | فعال یا غیرفعال بودن PHPMyAdmin
 |mysql_root_password| string| root| رمز عبور یوزر root دیتابیس|
+|volume_name| None| root| نام volumeای که به سرویس وصل می شود|
+
+> توجه داشته باشید که اگر میخواهید سرعت I/O در سرویس دیتابیس شما بیشتر شود٬ می‌توانید از volume‌ها استفاده کنید و نام آن را به صورت `c volume_name=VOLUME_NAME-` موقع ساخت Managed Service در cli وارد نمایید. ( VOLUME_NAME نام volume‌ای است که موقع ساخت آن تعیین کرده‌اید).
```

Comment:
c volume_name=VOLUME_NAME- ? 
does this show correctly in the rendered page?

Reply:
Render is ok, but there was a miss-place bug that is solved